akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala

```diff
@@ -21,16 +21,18 @@ class TimeoutsSpec extends AkkaSpec {
 
     "pass through elements unmodified" in assertAllStagesStopped {
       Await.result(
-        Source(1 to 100).via(Timeouts.initalTimeout(2.seconds)).grouped(200).runWith(Sink.head),
+        Source(1 to 100).initialTimeout(2.seconds).grouped(200).runWith(Sink.head),
         3.seconds) should ===(1 to 100)
     }
 
     "pass through error unmodified" in assertAllStagesStopped {
       a[TE] shouldBe thrownBy {
         Await.result(
-          Source(1 to 100).concat(Source.failed(TE("test")))
-            .via(Timeouts.initalTimeout(2.seconds))
-            .grouped(200).runWith(Sink.head),
+          Source(1 to 100).
+            concat(Source.failed(TE("test"))).
+            initialTimeout(2.seconds).
+            grouped(200).
```

Comment:
Thanks :)

-- 
Cheers,
Konrad 'ktoso’ Malawski
Akka @ Typesafe

On 1 November 2015 at 19:52:52, Viktor Klang (√) (notifications@github.com) wrote:

In akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala:

>          3.seconds) should ===(1 to 100)
>      }
>   
>      "pass through error unmodified" in assertAllStagesStopped {
>        a[TE] shouldBe thrownBy {
>          Await.result(
> -          Source(1 to 100).concat(Source.failed(TE("test")))
> -            .via(Timeouts.initalTimeout(2.seconds))
> -            .grouped(200).runWith(Sink.head),
> +          Source(1 to 100).
> +            concat(Source.failed(TE("test"))).
> +            initialTimeout(2.seconds).
> +            grouped(200).
I addressed your formatting comment :)
…
—
Reply to this email directly or view it on GitHub.